ci/pipelines/pr.yml

```diff
@@ -605,8 +605,7 @@ jobs:
     plan:
       - <<: *get-pull-request
         resource: card-connector-pull-request
-        trigger: false
-        passed: [card-connector-unit-test]
+        trigger: true
```

Comment:
We should probably be consistent and either have `trigger: true` explicitly written out for all the tasks (not just here), or remove this one?

Reply:
Removed 👍